modules/post/linux/gather/haserl_read.rb

```diff
@@ -0,0 +1,91 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::File
+  include Msf::Post::Linux::System
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Haserl Arbitrary File Reader',
+        'Description' => %q{
+          This module exploits haserl to read arbitrary files.
+          The most widely accepted exploitation vector is reading /etc/shadow,
+          which will reveal root's hash for cracking.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'Julien (jvoisin) Voisin', # metasploit module
+          'Ike Broflovski' # discovery
+        ],
+        'Platform' => [ 'linux' ],
+        'SessionTypes' => [ 'shell', 'meterpreter' ],
+        'References' => [
+          ['URL', 'https://twitter.com/steaIth/status/1364940271054712842'],
```

Comment:
```
You’re unable to view this Tweet because this account owner limits who can view their Tweets. Learn more
```

Reply:
Unfortunately, there is nothing I can do here :/